ios/FluentUI/Extensions/UIView+Extensions.swift

```diff
@@ -200,3 +200,17 @@ extension UIView {
         NSLayoutConstraint.contain(view: view, in: self, withInsets: insets, respectingSafeAreaInsets: respectsSafeAreaInsets)
     }
 }
+
+// MARK: - Pointer Interaction APIs check
+
+extension UIView {
+    
+    /// Some beta versions of iOS 13.4 and later don't include the pointer interaction APIs.
+    /// Checking the UIButton's isPointerInteractionEnabled property is a workaround that prevents a crash due to an
+    /// exception thrown while trying to add the UIPointerInteraction instance while running on those iOS versions:
+    /// "uncaught ObjC exception, reason: Invalid parameter not satisfying: interaction"
+    /// - Returns: true if Pointer Interaction APIs are available, otherwise false
+    func arePointerInteractionAPIsAvailable() -> Bool {
+        return UIButton.instancesRespond(to: Selector(("isPointerInteractionEnabled")))
+    }
+}
```

Comment:
Does this need to be an extension on UIView? Especially since it doesn't complement or use any of the built-in UIView behavior can we just have a standalone func?

Reply:
The reasoning behind putting it on a UIView extension is because the addInteraction function is also in a UIView extension. 
But I agree that a standalone function is a better approach.